packages/overlay/src/active-overlay.ts

```diff
@@ -133,6 +141,12 @@ export class ActiveOverlay extends LitElement {
 
     @property({ reflect: true })
     public placement: Placement = 'bottom';
+    @property({ attribute: false })
+    public color: Color = DefaultColor;
+    @property({ attribute: false })
+    public size: Size = DefaultSize;
+    @property({ type: Boolean, attribute: false })
+    public hasTheme = false;
```

Comment:
The `hasTheme` property allows me to be more strict about the typing of `size` and `color` and prevents the need to recast those when putting them into the `<sp-theme>` element later. I could be convinced that it's better to lean more heavily on the typing, however.